corehq/ex-submodules/casexml/apps/case/cleanup.py

```diff
@@ -73,6 +72,12 @@ def safe_hard_delete(case):
 
     This is used primarily for cleaning up system cases/actions (e.g. the location delegate case).
     """
+    if not settings.UNIT_TESTING:
+        from corehq.apps.commtrack.const import USER_LOCATION_OWNER_MAP_TYPE
+        whitelist_case_types = [USER_LOCATION_OWNER_MAP_TYPE]
+        if case.type not in whitelist_case_types:
```

Comment:
Is there a plan to add more types to this list? If not, how about `case.type == USER_LOCATION_OWNER_MAP_TYPE`?